config/locales/es.yml

```diff
@@ -205,6 +205,18 @@ es:
         pledge_sent: Promesa Enviada
         pledge_unfulfilled: Promesa No Cumplida
         resolved: Resuelto Sin %{fund}
+  flash:
+    account_locked: Cuenta actualmente bloqueada, consulte con su fondo.
+    clinic_created: %{clinic} creado!
+    error_saving_clinic: "Los errores evitaron que esta clínica se guardara: %{error}"
+    error_saving_clinic_details: "Error al guardar los detalles de la clínica: %{error}"
+    clinic_details_updated: Detalles de la clínica exitosamente actualizados
+    config_update_success: Configuración actualizada correctamente
+    config_failed_update: Config no pudo actualizar
+    new_patient_save: Un nuevo paciente ha sido salvado con éxito
+    new_patient_error: "Los errores impidieron salvar a este paciente: %{error}"
```

Comment:
Algúnos errores impidieron guardar a este paciente: %{error}

Reply:
In: 7e6d05f